locales/it/index.yml

```diff
@@ -787,3 +793,4 @@ reminders:
   loadMoreData: carica le scadenze del periodo precedente
 openMaps:
   genericError: Si è verificato un errore durante l'apertura della mappa
+genericError: Si è verificato un errore
```

Comment:
You could shift this alert under `general`

Reply:
I can't find `general`, do you mean `global`?

Reply:
yes!

Reply:
Ok, genericError is already under global